src/Service.Tests/Configuration/RuntimeConfigLoaderTests.cs

```diff
@@ -30,4 +33,63 @@ public async Task CanLoadStandardConfig(string configPath)
 
         Assert.IsTrue(loader.TryLoadConfig("dab-config.json", out RuntimeConfig _), "Failed to load config");
     }
+
+    [DataTestMethod]
+    [DataRow("Multidab-config.CosmosDb_NoSql.json", new string[] { "Multidab-config.MsSql.json", "Multidab-config.MySql.json", "Multidab-config.PostgreSql.json" })]
+    public async Task CanLoadValidMultiSourceConfig(string configPath, IEnumerable<string> dataSourceFiles)
+    {
+        string fileContents = await File.ReadAllTextAsync(configPath);
+
+        // Parse the base JSON string
+        JObject baseJsonObject = JObject.Parse(fileContents);
+
+        // Create a new JArray to hold the values to be appended
+        JArray valuesToAppend = new(dataSourceFiles);
+
+        // Add or append the values to the base JSON
+        baseJsonObject.Add("data-source-files", valuesToAppend);
+
+        // Convert the modified JSON object back to a JSON string
+        string resultJson = baseJsonObject.ToString();
+
+        IFileSystem fs = new MockFileSystem(new Dictionary<string, MockFileData>() { { "dab-config.json", new MockFileData(resultJson) } });
+
+        FileSystemRuntimeConfigLoader loader = new(fs);
+
+        Assert.IsTrue(loader.TryLoadConfig("dab-config.json", out RuntimeConfig runtimeConfig), "Should successfully load config");
+        Assert.IsTrue(runtimeConfig.ListAllDataSources().Count() == 4, "Should have 4 data sources");
+    }
+
+    [DataTestMethod]
+    [DataRow("dab-config.CosmosDb_NoSql.json", new string[] { "dab-config.MsSql.json", "dab-config.MySql.json", "dab-config.PostgreSql.json" })]
+    public async Task FailLoadMultiDataSourceConfigDuplicateEntities(string configPath, IEnumerable<string> dataSourceFiles)
```

Comment:
in a test comment, can you note how the entity is duplicated across config files? i.e. sql/mysql have an entity named "EntityA" causing failure. My point is raising this is to be specific about which file(s) have the duplicate. Perhaps all three files have the duplicate?
What is the user experience here, is the specific entity logged to the console?

Reply:
The specific entity is not logged to the console. I thought that would come as part of the CLI pr where the various runtimeconfig is generated and the runtimeconfig as a whole is validated. over here it will error out with this error: 
<img width="523" alt="image" src="https://github.com/Azure/data-api-builder/assets/124841904/e6f2ce6b-497b-4c07-83d5-c967f6833179">